Dockerfile

```diff
@@ -31,18 +32,20 @@ RUN git clone https://github.com/cloudera/kudu.git $KUDU_HOME && \
 
 VOLUME /data/kudu-master /data/kudu-tserver
 
-
 ####################
 # PORTS
 ####################
 #
 # https://github.com/cloudera/kudu/blob/master/docs/installation.adoc
-# 
+#
 # TabletServer:
-#	 7050 = TabletServer RPC Port
-# 	 8050 = TabletServer Web UI
+#   7050 = TabletServer RPC Port
+#   8050 = TabletServer Web UI
 # Kudu Master:
-#	 7051 = Master RPC Port
-#	 8051 = Master Web UI
+#   7051 = Master RPC Port
+#   8051 = Master Web UI
 #
-EXPOSE 8050 8051
+
```

Comment:
Maybe, it's ok to keep EXPOSE instruction since "docker run --link" could provide port sharing without specifying wish port we want. For sure, this is "Bells and whistles" and it don't break anything.